submarine-dist/src/assembly/distribution.xml

```diff
@@ -148,13 +148,6 @@
         <include>submarine-python-interpreter-${project.version}-shade.jar</include>
       </includes>
     </fileSet>
-    <fileSet>
-      <directory>../submarine-workbench/interpreter/spark/scala-2.11</directory>
-      <outputDirectory>/lib/interpreter/spark/scala-2.11</outputDirectory>
-      <includes>
-        <include>spark-scala-2.11-${zeppelin.version}.jar</include>
-      </includes>
-    </fileSet>
```

Comment:
Can `spark-scala-2.11-${zeppelin.version}.jar` be included in `spark-interpreter-${zeppelin.version}-shade.jar` ?

Reply:
Yes, All three Spark interpreters(2.10, 2.11, 2.12) have been packaged into the spark-interpreter-${zeppelin.version}-shade.jar. No more System Variable  like SUBMARINE_HOME or ZEPPELIN_HOME

Reply:
Cool!